### DISCUSSION TITLE
minio on nfs mount

### DISCUSSION THREAD

**Discussion by rstanonik** [minio/minio#15818] [Q&A] [answered]
We've used minio for years to provide s3 access to nfs mounts, but recently it seems to have stopped working.  As I recall minio has always warned against using nfs mounts.  Is that no longer allowed?  Any alternatives for providing s3 access to nfs mounts?

Thanks,
Ron

**Comment by harshavardhana** [comment, marked as answer]
@rstanonik we deprecated NAS gateway if you have an existing it will continue to work no fresh deployments are allowed. If you want extended support for your deployment buy a subscription with us https://min.io/pricing
